src/render_graph.rs

```diff
@@ -1,18 +1,37 @@
+use std::borrow::Cow;
+
 use crate::{
     dot::{font_tag, DotGraph},
     utils,
 };
 use bevy::render::render_graph::{Edge, NodeId, RenderGraph};
+use itertools::{EitherOrBoth, Itertools};
+
+/// Escape tags in such a way that it is suitable for inclusion in a
+/// Graphviz HTML label.
+pub fn escape_html<'a, S>(s: S) -> Cow<'a, str>
```

Comment:
There is already an `escape_html` function in `src/dot.rs`, you can use (and extend) that

Reply:
Done. I added escaping for `&` and `\"` as I already ran into issues with it.